website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -92,12 +78,28 @@ controllers/memcached_controller.go
 
 This will scaffold the Memcached resource API at `api/v1alpha1/memcached_types.go` and the controller at `controllers/memcached_controller.go`.
 
-See the [API terminology doc][api_terms_doc] for details on the CRD API conventions.
+**Note:** This guide will cover the default case of a single group API. If you would like to support Multi-Group APIs see the [Single Group to Multi-Group][multigroup-kubebuilder-doc] doc.
 
-To understand the API Go types and controller scaffolding see the Kubebuilder [api doc][kb_api_doc] and [controller doc][kb_controller_doc].
+#### Understanding API’s
+
+The goal of this command is to create Custom Resource (CR) and Custom Resource Definition (CRD) resources for the Memcached Kind. This command is creating the API with the group `cache.example.com`, and version `v1alpha1`  which uniquely identifies the new CRD of the Memcached Kind. For more info see [Extend the Kubernetes API with CustomResourceDefinitions][[kubernetes-extend-api]]. 
+
+Consequently, by using the Operator SDK tool, we can create our APIs and objects that will represent our solutions on these platforms. Here will adds only a single kind of resource; however, it could have as many Kinds as needed (1…N). Basically, the CRDs are a definition of our customized Objects, and the CRs are an instance of it.
+
+#### Getting a better idea
+
+Let’s think about the classic scenario where the goal is to have an application and its database running on the platform with Kubernetes. Then, one object could represent the App, and another one could represent the DB. By having one CRD to describe the App and another one for the DB, we will not be hurting concepts such as encapsulation, the single responsibility principle, and cohesion. Damaging these concepts could cause unexpected side effects, such as difficulty in extending, reuse, or maintenance, just to mention a few.
+
+In conclusion, the App CRD will have as its controller the DB CRD. Imagine, that a Deployment and Service are required for the application run so that the App’s Controller will provide these resources in this example. Similarly, the DB’s controller will have the business logic implementation of its objects.
```

Comment:
I'm not really sure what this paragraph is trying to say. The App CRD controller definitely isn't the DB CRD, as that's just another entry in the CRD table.

Reply:
>  The App CRD controller definitely isn't the DB CRD, as that's just another entry in the CRD table.

Sorry, I am not sure if I could understand your comment. However, this paragraph explains how the API works and adds an example where we would be creating 2 APIs. 1 that represents the App and another that represents the DB. So, if it still not clear could you please describe what is confusing you?

Reply:
"In conclusion, the App CRD will have as its controller the DB CRD" doesn't make any sense, as the App CRD controller would be some program you deploy to watch etcd, not the DB CRD. And the rest of the paragraph trails off from there. I think you're trying to say something like "For example, we would create a CRD to represent the App, and deploy a controller to watch and reconcile changes in that CRD. This controller would be responsible for things like creating Deployments that contain the App and creating Services to access it. Similarly, we would create a CRD to represent the DB, and deploy a controller that would manage DB instances."

Reply:
addressed your suggestion